release/lts/centos7/test-deps/meta.json

```diff
@@ -10,6 +10,6 @@
         "test-deps"
     ],
     "TestProperties": {
-        "size": 575
+        "size": 610
```

Comment:
Anything in particular that caused these jumps?

Reply:
probably just 7.2... there is also a 20% grace in the test... so, it takes a bit before we must bump the value.

Reply:
There were some cases where images had grown by large amounts. say powershell was put on the image twice... that's what we are looking for... "natural" growth doesn't worry me.